sdk/storage/azure-storage-blob/src/test/java/com/azure/android/storage/blob/StorageBlobClientTest.java

```diff
@@ -932,6 +933,127 @@ public void onFailure(Throwable throwable, Response response) {
         awaitOnLatch(latch, "deleteWithResponse");
     }
 
+    @Test
+    public void deleteContainer() {
+        // Given a StorageBlobClient.
+
+        // When deleting a blob using delete().
+        MockResponse mockResponse = new MockResponse()
+            .setResponseCode(202);
+
+        mockWebServer.enqueue(mockResponse);
+
+        // Then a response without body and status code 202 will be returned by the server.
+        Void response = storageBlobClient.deleteContainer("container");
+
+        assertNull(response);
+    }
+
+    @Test
+    public void deleteContainer_withCallback() {
+        // Given a StorageBlobClient.
+
+        // When deleting a blob using delete() while providing a callback.
+        MockResponse mockResponse = new MockResponse()
+            .setResponseCode(202);
+
+        mockWebServer.enqueue(mockResponse);
+
+        CountDownLatch latch = new CountDownLatch(1);
+
+        storageBlobAsyncClient.deleteContainer("container",
+            new CallbackWithHeader<Void, ContainerDeleteHeaders>() {
+                @Override
+                public void onSuccess(Void result, ContainerDeleteHeaders header, Response response) {
+                    try {
+                        // Then a response without body and status code 202 will be returned by the server to the callback.
+                        assertNull(response);
```

Comment:
Is the response supposed to be null? I feel like you should be able to check the code and such?

Reply:
Ah yeah my bad